app/controllers/pull_requests_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PullRequestsController < ApplicationController
+  layout 'sidebar_metrics'
+
+  def index
+    @pull_requests = Builders::Distribution::PullRequests.call(
+      department_name: params[:department_name],
+      from: metric_params[:period],
+      langs: metric_params[:lang] || []
+    )
+  end
+
+  def metric_params
+    params.require(:metric).permit(:period, lang: [])
```

Comment:
what's the difference with
```suggestion
    params.require(:metric).permit(:period, :lang)
```
?

Reply:
the `lang` param is an array of strings, if we keep it like `:lang` it gets excluded from the permitted scope